scvi/dataloaders/_data_splitting.py

```diff
@@ -95,13 +98,15 @@ def __init__(
         adata_manager: AnnDataManager,
         train_size: float = 0.9,
         validation_size: Optional[float] = None,
+        shuffle: bool = True,
```

Comment:
actually can you expose this in unsupervisedtrainingmixn?

Reply:
and totalvi.train, peakvi.train, multivi.train, JaxTrainingMixin.train, destvi.train, condscvi.train,

Reply:
cellassign.train, scbasset.train, and then check solo and stereoscope

Reply:
also could rename to `shuffle_set_split` as `shuffle` typically refers to minibatching